README.md

```diff
@@ -55,30 +55,29 @@ stack uninstall
 
 Learn about `normalize-phi` options.
 
-```sh
-normalize-phi --help
+```$ as console
```

Comment:
I'm not familiar with this syntax for fenced code blocks. What does it mean?

Reply:
It's for mdsh. mdsh will run the command(s) and write output to README below the command. I made a PR to the original repo yesterday (https://github.com/zimbatm/mdsh/pull/59). Also, one can run `nix run .#mdsh` in our repo.